### ISSUE TITLE
How to proceed with generated (and other) ViewHelper reference?

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/T3DocTeam#52]
* Where can you give feedback? Can we create a GitHub repo that has the glue pages and where issues can be created?I
* What should be done with Extbase / Fluid Guide? Does new generated VH reference contain all information, the Extbase / Fluid Guid VH ref did? If not, how do we get the info in there?
* Have the reservations against using reST for code snippets etc. been resolved?
* Should VH ref in Extbase / Fluid guide be taken down? If not now, what is the plan?

**Comment by sypets** [comment]
IMHO, what is missing right now, is a place to give feedback on generated VH ref. I propose to create a GitHub repo for the glue pages, so people can add issues there. 

See also: https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/issues/48

**Comment by sypets** [comment]
> MHO, what is missing right now, is a place to give feedback on generated VH ref. I propose to create a GitHub repo for the glue pages, so people can add issues there.

A place to report issues for generated VH ref now exists: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-ViewHelper/issues

I think, what is still missing is:

* Fixing the formatting for fluidtypo3. I created a new issue for that: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-ViewHelper/issues/4
* Documenting how to contribute. I created a PR for that: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-ViewHelper/pulls But the main information about contributing should probably go in Contribution Guide for core and contribution Guide in typo3fluid.
* Comparing "old" and "new" VH ref.  

Am closing this now.
